website/archiver/__init__.py

```diff
@@ -0,0 +1,73 @@
+ARCHIVER_INITIATED = 'INITIATED'
+ARCHIVER_FAILURE = 'FAILURE'
+ARCHIVER_SUCCESS = 'SUCCESS'
+ARCHIVER_SENT = 'SENT'
+
+ARCHIVER_PENDING = 'ARCHIVING'
+ARCHIVER_CHECKING = 'CHECKING'
+ARCHIVER_SENDING = 'SENDING'
+
+ARCHIVER_NETWORK_ERROR = 'NETWORK_ERROR'
+ARCHIVER_SIZE_EXCEEDED = 'SIZE_EXCEEDED'
+ARCHIVER_UNCAUGHT_ERROR = 'UNCAUGHT_ERROR'
+
+ARCHIVER_FAILURE_STATUSES = {
+    ARCHIVER_FAILURE,
+    ARCHIVER_NETWORK_ERROR,
+    ARCHIVER_SIZE_EXCEEDED,
+    ARCHIVER_UNCAUGHT_ERROR,
+}
+
+class StatResult(object):
+    """
+    Helper class to collect metadata about a single file
+    """
+    num_files = 1
+
+    def __init__(self, target_id, target_name, disk_usage=0):
+        self.target_id = target_id
+        self.target_name = target_name
+        self.disk_usage = float(disk_usage)
+
+    def __str__(self):
+        return str(self._to_dict())
+
+    def _to_dict(self):
```

Comment:
You may want to override `__dict__` or just use the default implementation of it which will probably perform the same way.